interfaces/vrep/robots/YouBotVrepRobot.m

```diff
@@ -17,11 +17,11 @@
 %       will become "youBot#0", a third robot, "youBot#1", and so on.
 %
 %   YouBotVrepRobot Methods:
-%       send_q_to_vrep - Sends the joint configurations to VREP
-%       get_q_from_vrep - Obtains the joint configurations from VREP
+%       set_configuration_space_positions - Sends the joint configurations to VREP
+%       get_configuration_space_positions - Obtains the joint configurations from VREP
 %       kinematics - Obtains the DQ_Kinematics implementation of this robot
 
-% (C) Copyright 2020 DQ Robotics Developers
+% (C) Copyright 2018-2024 DQ Robotics Developers
```

Comment:
2011-2024

Reply:
Updated in [badb11a](https://github.com/dqrobotics/matlab/pull/108/commits/badb11aa7d5893182e154ae0c97da61740393e5b).